server/index.js

```diff
@@ -3,14 +3,18 @@
 import express from "express";
 import "dotenv/config";
 
-import bridgeRouter from "./routes/v1/bridge.js";
-import swapRouter from "./routes/v1/swap.js";
+import bridgeRouter from "./routes/beta/bridge.js";
+import swapRouter from "./routes/beta/swap.js";
+import v1Router from "./routes/v1/index.js";
 
 const app = express();
 const port = process.env.port ?? 8080;
 
-app.use("/api/v1/bridge", bridgeRouter);
-app.use("/api/v1/swap", swapRouter);
+app.use("/beta/bridge", bridgeRouter);
```

Comment:
curious, why did you change name of these routes from `/api/v1` => `/beta`

Reply:
Moved the old api to `beta` cause I didn't want it to conflict with my v1. I'll get rid of it once I replicate all the functionality